### ISSUE TITLE
How to export?

### ISSUE THREAD

**Issue by marzof** [folkertdev/freestyle-svg-exporter#1]
Hi,
I really appreciate your work about this addon but (sorry for the dumb question) I don't find how to export.
I installed the plugin and it appears correctly. I checked it and made a rendering with freestyle.
After that I don't know how to get a svg file (I tried to save the result as image but there is no svg option among available formats).
Thanks,
Marco

**Comment by folkertdev** [comment]
hello Marzof,

The svg output follows blender's output (if you have a somewhat recent version). so, where the render result is saved to, for instance /tmp/test0000.png, the .svg is saved as /tmp/test0000.svg

hope this helps,
Folkert

**Comment by marzof** [comment]
Thanks Folkert,
I get it now.
Actually I found the svg file in /tmp folder: the same folder set as temporary path in Blender preferences (I'm on Ubuntu).
Now it's ok for me and I thank you (and the other developers) so much for this plugin.
Marco

PS: Have you planned to add, in the future, a field where users can set a custom path for the svg output? I think it could be helpful.

**Comment by folkertdev** [comment]
It's funny that you say that. The main reviewers of this project, Campbell Barton and TK advised to remove the seperate svg export path. In what way do you think exporting the .svg to another location than the render result could be useful?

**Comment by marzof** [comment]
I don't think it's a bad idea to export the .svg along with the render result and if I found the .svg file on the same folder where I saved my render image I would have never suggest a change (I'm not a programmer). But that's not what happened to me: actually if I save my render by F3 button (or by Image -> Save As Image menu) the software ask me to chose a location and, after that, only the raster image is being there. On the other hand, under my /tmp directory I get only the .svg files and no raster file at all.
Maybe I make some mistake in the process but, as far as I can see, it's not so obvious for users going and checking their own /tmp directory to find the render result.
